pkg/kube/misc.go

```diff
@@ -47,11 +47,17 @@ func GetServiceInterfaceTarget(targetType string, targetName string, deducePort
 			return nil, fmt.Errorf("Could not read deployment %s: %s", targetName, err)
 		}
 	} else if targetType == "statefulset" {
+		targetSvc, err := GetService(targetName, namespace, cli)
```

Comment:
Is this right? Should it not take the service name from the statefulset spec?

Reply:
Indeed, thanks!